src/cpp/session/modules/SessionRmdNotebook.R

```diff
@@ -608,37 +614,76 @@ assign(".rs.notebookVersion", envir = .rs.toolsEnv(), "1.0")
    .rs.fromJSON(.rs.base64decode(encoded))
 })
 
+.rs.addFunction("parseYamlOpt", function(opt)
+{
+    opt <- sub("^#\\|\\s*", "", opt)
+    if (grepl(".+=.+", opt)) # R-style chunk options
+        opts <- knitr:::parse_params(opt)
```

Comment:
Oops, need to update this to check for comments

Reply:
Actually, @kevinushey inline comments don't seem to be accepted by knitr either, so I'm not sure that we need to support comments. Where did you see that inline comments starting with `#` would be supported?

Reply:
However, multiline chunk options are supposed to be supported, so I think I need to add that support here

```
#| warning = F, message = F, fig.width = 8,
#| fig.cap = "This is a long long
#|   long long caption."
```

Reply:
Okay, added support for multiline, but not for comments, which I don't think are actually meant to be supported

Reply:
I was using this as an example:

`````
---
title: "Use R Markdown"
output: html_document
---

```{r}
#| eval: false  # this is a comment
print("Hello, world!")
```
`````

When I render this R Markdown document, the `eval: false` option is parsed and read, and the associated chunk is not evaluated.

```
label: unnamed-chunk-1 (with options) 
List of 1
 $ eval: logi FALSE
```

Reply:
Weird because I tried that and got an error... I'll check again

Reply:
Oh I think I tried it with the R style chunk options, `#| eval = FALSE # this is a comment`

Reply:
Okay -- so in-line comments work with YAML format but not with R code format. This is true both when rendered and when running code in the console (with run all chunks above). No need for specific further changes to the code to support the inline comments but will make sure to add that to QA notes